docs/using.rst

```diff
@@ -313,9 +313,16 @@ handled by a single executable that is registered in the bundle's manifest:
   [hooks]
   filename=hook
 
-Each hook must be activated explicitly and leads to a call of the hook executable
-with a specific argument that allows to distinguish between the different hook
-types. Multiple hook types must be separated with a ``;``.
+The ``filename`` must match the name of the script or binary executable placed
+inside the content folder the bundle is generated from.
+
+The actual hook invocation must be registered explicitly in the manifest via
```

Comment:
```suggestion
The actual hook invocation must be registered for each `[image.*]` in the manifest via
```

Reply:
I had this first, but this would exclude "bundle hooks" (i.e. install-check) which are not inside an `image` section. So the other alternative would be to mention both. `[image.*]` and `[hooks]`.